app/sign.js

```diff
@@ -131,8 +96,9 @@ const getBackupSigningKey = function(key, expectedXpub) {
 }
 
 const handleSignUtxo = function(recoveryRequest, key, skipConfirm) {
-  const network = utxoNetworks[recoveryRequest.coin];
-  const decimals = coinDecimals[recoveryRequest.coin];
+  
+  const network = getCoinConfig(recoveryRequest.coin, network);
```

Comment:
Are `network` and `decimalPlaces` defined here? Maybe you wanted to use string keys.

Reply:
Thanks for notice. Can you check if the lastest commit is legit? Haven't coded in JS for a long time so need to catch up its syntax.
https://github.com/BitGo/key-recovery-service-v2/pull/65/commits/cb650e885ba34d92d6c8074b06bc7515253d669d